.storybook/main.js

```diff
@@ -1,6 +1,5 @@
-const path = require('path');
+const path = require('node:path');
```

Comment:
I'm having trouble finding what Node version the `node:path` syntax was introduced. It might not work with Node 14, but I'll need to test.

Reply:
https://www.wisdomgeek.com/development/web-development/javascript/node-js-introduces-node-protocol-imports/

It was introduced in node 14.18